scripts/distro-configure.js

```
@@ -12,13 +13,27 @@ var ENVS = { development: true, production: true }
 
 forceNodeEnvProduction()
 
+function getReleasePlatform(){  
+  switch (os.platform()) {
+    case 'darwin':
+      return 'mac'
+    case 'win32':
+      return 'windows'
+    case 'linux':
+      return 'linux'
+    default:
+      return 'unknown'
```

Ideally, we should throw an error here to prevent any callers from proceeding.

---

:+1: